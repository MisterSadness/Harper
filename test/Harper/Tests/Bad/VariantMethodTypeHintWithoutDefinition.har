value HelloWorld = {
    variant Hi = {
      sayHello :: String;
      sayHello = "Hi, World!";
    };

    variant Hello = {
      sayHello :: String;
    };
  };