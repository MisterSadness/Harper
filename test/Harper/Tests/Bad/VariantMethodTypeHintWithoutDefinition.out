Error: type hint for function `sayHello` has no associated definition.
During evaluation of: 
    sayHello :: String 
    
    Located at line 8 column 7 
    
Execution terminated with an error: type error.